### PR TITLE
OpenID: Add claims to Well know

### PR DESCRIPTION
**Pull request by AndreaGreco**
## Description of the Change

Some application, don't use user_info, and read directly by wellknow
Add to wellknow related key and changed class for support list and custom claims

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [x] unit-test added
- [x] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

**Review by auvipy — APPROVED** [review]

**Comment by n2ygk** [comment]
@AndreaGreco I've been forced to revert this PR because you changed the signature of `get_additional_claims` by dropping the `request` parameter. See #1066 for more details. The original reviewer should not have merged it.

Please review and resubmit a PR that doesn't change the signature and I'll be happy to review it. Thanks.

**Comment by Natureshadow** [comment]
Also, please consider my comment: https://github.com/jazzband/django-oauth-toolkit/pull/1068#issuecomment-1005127680

File: docs/oidc.rst [hunk 1]
```diff
@@ -245,16 +245,17 @@ required claims, eg ``iss``, ``aud``, ``exp``, ``iat``, ``auth_time`` etc),
 and the ``sub`` claim will use the primary key of the user as the value.
 You'll probably want to customize this and add additional claims or change
 what is sent for the ``sub`` claim. To do so, you will need to add a method to
-our custom validator::
-
+our custom validator.
+Standard claim ``sub`` is included by default, for remove it override ``get_claim_list``::
```
[the remark this answers is not in the document]
Comment (AndreaGreco) [reply docs/oidc.rst:249]: This is StandardClaims, some application use it for discover available params.
My case application is https://docebo.com

As commented inside code:
https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims

[the remark this answers is not in the document]
Comment (Natureshadow) [reply docs/oidc.rst:249]: The remark was about the preposition.

**Pull request merged**